Convenus/ExchangeServiceHelper.cs

```diff
@@ -284,5 +284,25 @@ public class CalendarEvent
         public DateTime EndTime { get; set; }
         public string Subject { get; set; }
 
+    }
+
+    public enum RoomStatus
+    {
+        /// <summary>
+        /// Cannot connect to Convenus or network error
+        /// </summary>
+        Red,

```

Comment:
I would explicitly number these `Red = 0`, `Green = 0`, etc since we are casting into an unsigned short in the service